contentgrid-spring-boot-autoconfigure/build.gradle

```diff
@@ -22,6 +22,7 @@ dependencies {
     annotationProcessor 'org.projectlombok:lombok'
 
     implementation platform(project(':contentgrid-spring-boot-platform'))
+    implementation project(':contentgrid-spring-boot-actuators')
```

Comment:
I believe this would be a `compileOnly` dependency, with `ActuatorAutoConfiguration` gated behind `@ConditionalOnClass`

Reply:
yes - typically the `-autoconfigure` project has no runtime dependencies, it's the `-starter` that pulls in the (runtime) dependencies